src/bin/wasmer.rs

```diff
@@ -492,6 +492,20 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
         None
     };
 
+    // Update backend when a backend flag is `auto` or a default value.
+    // Use a singlepass if it's enable and the file provided is larger
+    // than 10MiB (10485760 bytes), or it's enable and the target architecture
+    // is aarch64. Otherwise, use a cranelift as a backend.
+    if options.backend == Backend::Auto {
+        if Backend::variants().contains(&Backend::Singlepass.to_string())
+            && (wasm_binary.len() > 10485760 || cfg!(target_arch = "aarch64"))
```

Comment:
Based on my comprehension of #998, when the `--backend` option is set to `auto`, we want to use Cranelift by default, or Singlepass if the module is larger than 10MiB; which translates as:

```rust
if Backend::variants().containts(&Backend::Singlepass.to_string()) // Singlepass is available
    && wasm_binary.len() > 10485760 // Large module, be a good candidate for Singlepass
{
    options.backend = Backend::Singlepass;
} else {
    options.backend = Backend::Cranelift;
}
```

Reply:
The issue #998 only mentions the file size, but I heard from @syrusakbary on Slack that the  Singlepass backend is used when the target arch is AArch64. Is it better to add about it to the issue #998, or to remove the condition relating to AArch64 for now?